### PR TITLE
feat: upgraded nuxt and vue to latest versions

### PR DESCRIPTION
**Pull request by jeffdowdle**
**Issue**:

### What I did

- 
- 

### How to test

- Added `pathPrefix: false` to fix nuxt imports after updating
- Updated vue because nuxt required it
- Changed ripple-ui-forms to 'type: module' to match ripple-ui-core (nuxt complained about this after update)
- Changed ripple-ui-forms to use addComponentsDir to match ui-core

#### For all PR's

- [ ] I've added relevant changes to the project Readme if needed.
- [ ] I've updated the documentation site as needed.
- [ ] I have added unit tests to cover my changes (if not applicable, please state why in a comment)

#### For new components only

- [ ] I have added a story covering all variants
- [ ] I have checked a11y tab in storybook passes
- [ ] Any events are emitted on the event bus

### CODE REVIEW

**Comment by dylankelly** [comment]
@jeffdowdle can you also update https://github.com/dpc-sdp/ripple-framework/blob/develop/packages/nuxt-ripple-cli/src/commands/init/_templates/site/latest/package.json.t#L28 and https://github.com/dpc-sdp/ripple-framework/blob/develop/packages/nuxt-ripple-cli/src/commands/init/_templates/layer/latest/package.json.t#L34 ?

**Review by dylankelly — CHANGES_REQUESTED** [review]
@jeffdowdle can you also update https://github.com/dpc-sdp/ripple-framework/blob/develop/packages/nuxt-ripple-cli/src/commands/init/_templates/site/latest/package.json.t#L28 and https://github.com/dpc-sdp/ripple-framework/blob/develop/packages/nuxt-ripple-cli/src/commands/init/_templates/layer/latest/package.json.t#L34 ?

**Comment by jeffdowdle** [comment]
> @jeffdowdle can you also update https://github.com/dpc-sdp/ripple-framework/blob/develop/packages/nuxt-ripple-cli/src/commands/init/_templates/site/latest/package.json.t#L28 and https://github.com/dpc-sdp/ripple-framework/blob/develop/packages/nuxt-ripple-cli/src/commands/init/_templates/layer/latest/package.json.t#L34 ?

Done

**Review by dylankelly — APPROVED** [review]

File: packages/ripple-ui-core/src/nuxt/index.ts [hunk 1]
```diff
@@ -27,7 +27,8 @@ export default <any>defineNuxtModule({
     addComponentsDir({
```
Comment (dylankelly) [inline packages/ripple-ui-core/src/nuxt/index.ts:27]: can you add console message here to let people know its added?

**Review by lambry — APPROVED** [review]

**Pull request merged**